cli-output/src/cli_output.rs

```diff
@@ -3256,5 +3283,10 @@ mod tests {
         let s = format!("{c}");
         assert!(!s.is_empty());
         println!("{s}");
+
+        c.use_csv = true;
+        let s = format!("{c}");
+        assert!(!s.is_empty());
+        println!("{s}");
```

Comment:
think it's fine since other tests are also like this, but this isn't an automatic test.
We can pretty easily write to a string and check the value is what we expect, instead of println here. wdyt?

Reply:
yea, good idea. added